addons/bus/static/src/js/services/bus_service.js

```diff
@@ -62,7 +62,7 @@ var BusService =  CrossTab.extend(ServicesMixin, {
      * @param {string} content
      * @param {function} [callback] if given callback will be called when user clicks on notification
      */
-    sendNotification: function (title, content, callback) {
+    sendNotification: function (title, content, className, type, callback) {
```

Comment:
feels like those should just be `options` instead of specific positional parameter, similar to what is done in `displayNotification` in web